# Tools, workflows and pipelines

The following are links to Genomics Aotearoa GitHub repositories that inmplement specific 
analysis workflows.

- [environmental_metagenomics](https://github.com/GenomicsAotearoa/environmental_metagenomics) - A quick workflow to get researchers started with the bioinformatic pipeline used in the Environmental Metagenomics program.
- [long-read-microbial-isolate-genomics](https://github.com/GenomicsAotearoa/long-read-microbial-isolate-genomics) - Sequence processing workflow for samples obtained from microbial isolates sequenced using long-read technology.
- [High-quality-genomes](https://github.com/GenomicsAotearoa/High-quality-genomes) - This repository aims to hosting workflows/codes that have been developed for GA high-quality-genome projects.
- [data-management-resources](https://github.com/GenomicsAotearoa/data-management-resources) - Repo supporting the manuscript 'Journeying towards best practice data management in biodiversity genomics'.
- [Kakapo](https://github.com/GenomicsAotearoa/Kakapo) - Scripts and code for Kākāpō genomic data
- [BMSB-popgenomics](https://github.com/GenomicsAotearoa/BMSB-popgenomics) - Population genomic analysis of the brown marmorated stink bug, Halyomorpha halys for the GA Invasomics project
- [GraphPanGenomes_Microbial](https://github.com/GenomicsAotearoa/GraphPanGenomes_Microbial) - Microbial genomics pipeline to build and analyse genome graphs
- [vcf_annotation_pipeline](https://github.com/GenomicsAotearoa/vcf_annotation_pipeline) - A Snakemake workflow to filter, annotate and prepare variant call format (VCF) data for scout using GATK4, SnpSift, VEP and genmod. Designed to be used after human_genomics_pipeline.
- [human_genomics_pipeline](https://github.com/GenomicsAotearoa/human_genomics_pipeline) - A Snakemake workflow to process single samples or cohorts of paired-end sequencing data (WGS or WES) using trim galore/bwa/GATK4/parabricks.
